docs/source/model-registry.rst

```diff
@@ -78,47 +79,170 @@ version to one of the other valid stages.
 
 
 API Workflow
-------------
+============
+
+An alternative way to interact with Model Registry is using the `MLflow model flavor <https://www.mlflow.org/docs/latest/python_api/index.html>`_ or `MLflow Client Tracking API <https://www.mlflow.org/docs/latest/python_api/mlflow.tracking.html>`_ interface.
+In particular, you can register a model during an MLflow experiment run or after all your experiment runs.
+
+Adding an MLflow Model to the Model Registry
+--------------------------------------------
 
-You can register a model using two APIs: ``register_model`` and ``log_model``.
+There are three programmatic ways to add a model to the registry. First, you can use the ``mlflow.model_flavor.log_model(...)`` method. For example, in your code:
 
-If the model has already been logged, you can add it to the registry with the ``register_model`` API, referencing the model by its URI.
+.. code-block:: py
+
+    with mlflow.start_run(run_name="YOUR_RUN_NAME") as run:
+        params={"n_estimators": 5, "random_state": 42}
+        sk_learn_rfr=RandomForestRegressor(params)
+        # Log params using the MLflow sklearn flavor APIs
```

Comment:
What does this line mean? There is not a flavored param logger.

Reply:
I meant " Log parameters using the MLflow API"